newIDE/app/src/MainFrame/MainFrame.css

```diff
@@ -30,7 +30,7 @@
 :not(input):not(textarea):not(canvas)::before {
     -webkit-user-select: none;
     user-select: none;
-    cursor: default;
+    //cursor: default;
```

Comment:
What is the reason behind commenting this?

Reply:
For showing cursor with a "hand" onhover of button +
![image](https://user-images.githubusercontent.com/1670670/59977020-6f055c00-95cc-11e9-818e-f0eda14e246c.png)

Reply:
Better in video, look the cursor (On left actual beta/ on right this PR)
https://www.youtube.com/watch?v=wXRrFN3aP9s

Reply:
I'm not sure about this. Sure this is what is done on websites, but GDevelop is an application and should have the look'n'feel of an application: the fact that something can be clicked should be displayed by some hover effects - which is the case on "Click to add an animation" - not by a hand cursor which is too close to a "webpage link" feeling.

Reply:
ok i will uncomment this too